### ISSUE TITLE
Support for multikey secrets

### ISSUE THREAD

**Issue by alonisser** [Soluto/kamus#65]
Question: In k8 native secret I can have
name: secretsname:
key1:saaa,
key2:2222

Does kamus support this option?

**Comment by omerlh** [comment]
Kamus works a bit differently - and the documentation might not be good enough to explain that. Kamus let you encrypt and decrypt value, and does not enforce how you persist them. The simplest way is to work with [Kamus init container](https://github.com/Soluto/kamus/tree/master/init-container), which reads the encrypted values from a config map. This lets you put how many values you want :) Take a look at the [example app](https://github.com/Soluto/kamus/tree/master/example), which hopefully should make things clearer

**Comment by omerlh** [comment]
Yes, the init container create a configuration file into the shared volume with the decrypted values. The documentation *should* specify that currently there is support for JSON and CFG (and also support to create one file per key). So if you have the following config map:
```yaml
type: ConfigMap
metadata:
  name: test-cm
spec:
  key: <encrypted>
  key2: <encrypted>
```
Than the result will look like the following:
```json
{
  "key": "<decrypted>",
  "key2": "<decrypted>"
}
```
